apps/web-giddh/src/app/services/general.service.ts

```
@@ -19,6 +19,7 @@ import { HttpClient } from '@angular/common/http';
 import { IServiceConfigArgs, ServiceConfig } from './service.config';
 import { LedgerViewEnum } from '../models/api-models/Ledger';
 import { IOption } from '../theme/ng-virtual-select/sh-options.interface';
+import { giddhRoundOff } from '../shared/helpers/helperFunctions';
 import { AccountArchivedStatusEnum } from '../shared/Enums/common.enum';
 import { giddhRoundOff } from '../shared/helpers/helperFunctions';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'giddhRoundOff'. Consider to delete it or rename it.

'giddhRoundOff' is defined here:


(lint/suspicious/noRedeclare)